identity-service/src/notifications/sendNotifications/publishNotifications.js

```diff
@@ -97,6 +98,21 @@ const getPublishTypes = (userId, baseNotificationType, userNotificationSettings)
   return types
 }
 
+/**
+ * Checks if a notification type is enabled with optimizely
+ * @param {string} notificationType
+ * @param {*} optimizelyClient Optimizely client
+ */
+const shouldFilterNotification = (notificationType, optimizelyClient) => {
+  if (!optimizelyClient) {
```

Comment:
should this be true? are we saying here that we want to filter out if there is no optimizely client?

Reply:
oh hmm, seeing how this function is used, maybe you're right with the false, but we should probably name the function `shouldFilterNotification` instead, and also return the `!getFeatureFlag(optimizelyClient, FEATURE_FLAGS.REWARDS_NOTIFICATIONS_ENABLED)`